WordPressComKitTests/SiteServiceTests.swift

```diff
@@ -1,110 +1,55 @@
 import XCTest
-import OHHTTPStubs
-import WordPressComKit
+@testable import WordPressComKit
 
 class SiteServiceTests: XCTestCase {
-    var subject: SiteService!
+    let subject = SiteService()
```

Comment:
It's been my experience that a unit test should be re-instantiating the subject for every test to isolate each test from a previous one.